SpywareFilter/sections/whitelist.txt

```diff
@@ -26,6 +26,8 @@
 !#endif
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/111933
+@@||googletagmanager.com/gtm.js$domain=n11.com
```

Comment:
What is this rule for?

Reply:
@zloyden `googletagmanager.com/gtm.js` rule is blocking to click these.
![image](https://user-images.githubusercontent.com/35523759/156883285-0a8d5ba5-957c-4311-9c7d-8934253365c4.png)